### PR TITLE
Fix/ Remove features from web app

### PR DESCRIPTION
**Pull request by borkopetrovicc**
## Description

https://jwplayer.atlassian.net/jira/software/c/projects/PS/boards/575?selectedIssue=PS-242

This PR solves # .

### Steps completed:

According to our definition of done, I have completed the following steps:

- [ ] Acceptance criteria met
- [ ] Unit tests added
- [ ] Docs updated (including config and env variables)
- [ ] Translations added
- [ ] UX tested
- [ ] Browsers / platforms tested
- [ ] Rebased & ready to merge without conflicts
- [ ] Reviewed own code

### CODE REVIEW

File: packages/ui-react/src/components/Account/Account.tsx [hunk 1]
```diff
@@ -384,46 +384,6 @@ const Account = ({ panelClassName, panelHeaderClassName, canUpdateEmail = true }
                 </div>
               ),
             }),
-          canExportAccountData &&
-            formSection({
-              label: t('account.export_data_title'),
-              content: (section) => (
-                <div className={styles.textWithButtonContainer}>
-                  <div>
-                    <Trans t={t} i18nKey="account.export_data_body" values={{ email: section.values.email }} />
```
Comment (AntonLantukh) [inline packages/ui-react/src/components/Account/Account.tsx:393]: Do we still need translations for delete and export?

File: packages/ui-react/src/components/Account/Account.tsx [hunk 2]
```diff
@@ -72,7 +72,7 @@ const Account = ({ panelClassName, panelHeaderClassName, canUpdateEmail = true }
     shallow,
   );
 
-  const { canChangePasswordWithOldPassword, canExportAccountData, canDeleteAccount } = accountController.getFeatures();
+  const { canChangePasswordWithOldPassword, canExportAccountData } = accountController.getFeatures();
```
Comment (AntonLantukh) [inline packages/ui-react/src/components/Account/Account.tsx]: Do we still need "canExportAccountData"?

Comment (borkopetrovicc) [reply packages/ui-react/src/components/Account/Account.tsx]: 01a95c9ceda9cf3127a3fe29a2af2a8109642d2e

**Comment by AntonLantukh** [comment]
@borkopetrovicc it looks like tests are failing because of the code that was removed. Can you look into it?

**Comment by AntonLantukh** [comment]
@borkopetrovicc it looks looks good to me in general but I didn't work a lot with the payments logic. @ChristiaanScheermeijer any concerns you have here?

**Comment by ChristiaanScheermeijer** [comment]
I'm afraid this also removal doesn't take the Cleeng integration into consideration. We want to keep that logic intact as much as possible.

Also, when using Cleeng, I now see two cancel subscription buttons:

image

**Comment by borkopetrovicc** [comment]
> I'm afraid this also removal doesn't take the Cleeng integration into consideration. We want to keep that logic intact as much as possible.
> 
> Also, when using Cleeng, I now see two cancel subscription buttons:
> 
> image

What do you suggest ?

**Comment by borkopetrovicc** [comment]
> I'm afraid this also removal doesn't take the Cleeng integration into consideration. We want to keep that logic intact as much as possible.
> 
> Also, when using Cleeng, I now see two cancel subscription buttons:
> 
> image

8da8eadb1203eebfccc7b980f0abd52455b2706d

File: packages/ui-react/src/components/Account/Account.tsx [hunk 3]
```diff
@@ -72,7 +62,7 @@ const Account = ({ panelClassName, panelHeaderClassName, canUpdateEmail = true }
     shallow,
   );
 
-  const { canChangePasswordWithOldPassword, canExportAccountData, canDeleteAccount } = accountController.getFeatures();
+  const { canChangePasswordWithOldPassword } = accountController.getFeatures();
```
Comment (ChristiaanScheermeijer) [inline packages/ui-react/src/components/Account/Account.tsx:65]: This mechanism is in place to disable these features for specific integrations. Why would you remove this mechanism and implementation?

Just set `canExportAccountData` and `canDeleteAccount` to false and done, right? Custom integrations can still implement these features.

File: packages/ui-react/src/components/Payment/Payment.tsx [hunk 4]
```diff
@@ -219,78 +201,18 @@ const Payment = ({
...
-                  />
-                ))}
-              <div className={styles.changePlanButtons}>
-                <Button
-                  label={t('user:account.save')}
-                  onClick={onChangePlanClick}
-                  disabled={selectedOfferId === activeSubscription?.accessFeeId || changeSubscriptionPlan.isLoading}
-                />
-                <Button label={t('user:account.cancel')} onClick={() => setIsChangingOffer(false)} variant="text" />
-                {activeSubscription?.status !== 'cancelled' && (
```
Comment (ChristiaanScheermeijer) [inline packages/ui-react/src/components/Payment/Payment.tsx]: This removes the following features for **all** integrations:

- Downgrade/upgrade subscription
- Renew subscription when it has been cancelled
- Complete subscription when user is not subscribed

I don't think we want to do that (yet)... @dbudzins @AntonLantukh ?

**Review by ChristiaanScheermeijer — CHANGES_REQUESTED** [review]

**Comment by ChristiaanScheermeijer** [comment]
@dbudzins perhaps we could add some more description/definition to the PR descriptions like we do? That helps us to better understand the desired outcome and review this more efficient.

File: packages/ui-react/src/components/Payment/Payment.tsx (hunk 4 above)
Comment (borkopetrovicc) [reply packages/ui-react/src/components/Payment/Payment.tsx]: I will close these pr i've opened new pr

**Pull request closed without merging**